packages/calendar/src/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import {useState} from 'react'
+import getUserLocale from 'get-user-locale'
+import {useDefaults} from '@smashing/theme'
+import {CalendarProps, CalendarAppearanceType} from './types'
+import {StyledContainer, StyledInput, StyledCalendar} from './styles'
+
+const Calendar: React.FC<CalendarProps> = ({
+  onChange = (date: Date) => {},
+  value,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false)
+  const defaults = useDefaults('calendar', props, {
+    appearance: 'default' as CalendarAppearanceType
+  })
+
+  const getTwoDigitNumber = variable => {
+    return variable > 9 ? variable : '0' + variable
```

Comment:
Alternative:
```suggestion
    return String(variable).padStart(2, '0')
```

Reply:
I had one problem with padStart: I'm not sure what browsers are we targeting with Smashing-UI ;) If we're dropping <5% usage, then use padStart.

Reply:
I spoke with Kamil and we can use date format depending on user locale and I used `toLocaleDateString` function to format dates, so we haven't to worry about this ;)

Reply:
@curryy it's related to another comment i guess ;)